addons/website_sale/static/tests/tours/website_sale_cart_popover.js

```diff
@@ -0,0 +1,49 @@
+/** @odoo-module **/
+
+import tour from 'web_tour.tour';
+import wTourUtils from 'website.tour_utils';
+
+tour.register('website_sale_cart_popover_tour', {
+        test: true,
+        url: '/shop/chair-floor-protection-29',
+    },
+    [
+        wTourUtils.clickOnElement('Add to Cart', '#product_detail form[action^="/shop/cart/update"] #add_to_cart'),
+        {
+            content: "hover on cart popover",
+            trigger: '#top_menu a[href$="/shop/cart"]',
+            run: () => {
+                $('#top_menu a[href$="/shop/cart"]').mouseenter();
```

Comment:
Could not make work
```javascript
document.querySelector('#top_menu a[href$="/shop/cart"]').dispatchEvent(new Event('mouseenter'));
```

Reply:
Sometimes listeners don't work from jquery to vanilla and vice versa